traitsui/qt4/ui_panel.py

```diff
@@ -925,12 +925,12 @@ def _add_items(self, content, outer=None):
                     else:
                         item_height = max(item_height, height)
 
-                    control.setMinimumWidth(max(item_width, 0))
-                    control.setMinimumHeight(max(item_height, 0))
+                    control.setMinimumWidth(max(int(item_width), 0))
```

Comment:
looking at earlier code we already conver `item_height` to `int`. I think it is better if we change the core earlier in lines 916 and 926 to convert height and width into int

Reply:
Thanks - I'd missed the earlier explicit `int()`

Reply:
Don't we still need to apply int but at the width and height ariables in lines 916 and 926

Reply:
No, I think we're good. `width` and `height` are coming from `control.minimumSizeHint` so they are guaranteed to be ints, and `item_width` and `item_height` are being coerced.  Am I missing anything else?